src/app/core/setup/setup.service.ts

```diff
@@ -95,11 +95,11 @@ export class SetupService {
       Entity.extractTypeFromId(doc["_id"]),
     );
     if (!entityType) {
-      throw new Error("EntityType for entityToImport not found: " + doc["_id"]);
+      throw new Error("RecordType for recordToImport not found: " + doc["_id"]);
     }
 
     const entity = this.schemaService.loadDataIntoEntity(new entityType(), doc);
-    Logging.debug("Importing baseConfig entity", entity);
+    Logging.debug("Importing baseConfig record", entity);
```

Comment:
I think we should still use an entity instead of a record for logging errors and warnings, as it would make debugging easier.